packages/tempus/test/DIRK/Tempus_DIRK_VanDerPol.xml

```diff
@@ -101,7 +156,117 @@
+              </ParameterList>
+              <Parameter name="Method" type="string" value="Full Step"/>
+           </ParameterList>
+           <Parameter name="Nonlinear Solver" type="string" value="Line Search Based"/>
+           <ParameterList name="Printing">
+              <Parameter name="Output Precision" type="int" value="3"/>
+              <Parameter name="Output Processor" type="int" value="0"/>
+              <ParameterList name="Output Information">
+                 <Parameter name="Error" type="bool" value="1"/>
+                 <Parameter name="Warning" type="bool" value="1"/>
+                 <Parameter name="Outer Iteration" type="bool" value="0"/>
+                 <Parameter name="Parameters" type="bool" value="0"/>
+                 <Parameter name="Details" type="bool" value="0"/>
+                 <Parameter name="Linear Solver Details" type="bool" value="1"/>
+                 <Parameter name="Stepper Iteration" type="bool" value="1"/>
+                 <Parameter name="Stepper Details" type="bool" value="1"/>
+                 <Parameter name="Stepper Parameters" type="bool" value="1"/>
+              </ParameterList>
+           </ParameterList>
+           <ParameterList name="Solver Options">
+              <Parameter name="Status Test Check Type" type="string" value="Minimal"/>
+           </ParameterList>
+           <ParameterList name="Status Tests">
+              <Parameter name="Test Type" type="string" value="Combo"/>
+              <Parameter name="Combo Type" type="string" value="OR"/>
+              <Parameter name="Number of Tests" type="int" value="2"/>
+              <ParameterList name="Test 0">
+                 <Parameter name="Test Type" type="string" value="NormF"/>
+                 <Parameter name="Tolerance" type="double" value="1.0e-8"/>
+              </ParameterList>
+              <ParameterList name="Test 1">
+                 <Parameter name="Test Type" type="string" value="MaxIters"/>
+                 <Parameter name="Maximum Iterations" type="int" value="10"/>
+              </ParameterList>
+           </ParameterList>
+        </ParameterList>
+     </ParameterList>
+  </ParameterList>
+
+  <ParameterList name="SDIRK 2(1) Pair Embedded Stepper">
+     <Parameter name="Stepper Type" type="string" value="SDIRK 2(1) Pair"/>
+     <Parameter name="Use Embedded" type="bool" value="true"/>
+     <Parameter name="Solver Name"    type="string" value="Default Solver"/>
+     <ParameterList name="Default Solver">
+        <ParameterList name="NOX">
+           <ParameterList name="Direction">
+              <Parameter name="Method" type="string" value="Newton"/>
+              <ParameterList name="Newton">
+                 <Parameter name="Forcing Term Method" type="string" value="Constant"/>
+                 <Parameter name="Rescue Bad Newton Solve" type="bool" value="1"/>
+              </ParameterList>
+           </ParameterList>
+           <ParameterList name="Line Search">
```

Comment:
This is quite a long list of options. Is it possible to get away with more defaults to trim down the input files? I think you could remove the options, line search and direction sublists from the nox solver.